lib/Service/FederationService.php

```diff
@@ -113,7 +124,7 @@ public function isTrustedRemote($domainWithPort) {
 				break;
 			}
 			$regex = '/^' . implode('[-\.a-zA-Z0-9]*', array_map(fn ($v) => preg_quote($v, '/'), explode('*', $trusted))) . '$/i';
```

Comment:
Maybe we can add a comment so the next reader has it easier then us in our call pulling this into pieces to figure out again what it does ;)